### PR TITLE
Updated with Helmet and new movie genres

### PR DESCRIPTION
**Pull request by veerleprins**
Hi guys,

I've updated the code with Helmet HTTP header security and updated filter.ejs with some new movie genres to filter on! :)

### CODE REVIEW

File: routes/profile.js [hunk 1]
```diff
@@ -34,9 +34,8 @@ router.post('/forgotPassword', forgotPassword);
 async function profileOfMe(req, res, next) {
   // Rowan
   try {
-    let database = await usersCollection.find().toArray(); // this code can be removed at the point sessions works.
-    let myself = database.filter(showMe);
-    res.render('profile.ejs', { user: myself });
+    console.log(req.session.user);
+    res.render('profile.ejs', { user: req.session.user });
```
Comment (joordy) [inline routes/profile.js:38]: Ah good! Can we use req.session.user over the whole application now?

File: server.js [hunk 2]
```diff
@@ -19,6 +20,11 @@ const PORT = process.env.PORT || process.env.DB_PORT;
 // Middleware set-up
 app.set('view engine', 'ejs');
 app.set('views', 'view');
+app.use(helmet.frameguard({ action: 'deny' }));
+app.use(helmet.ieNoOpen());
+app.use(helmet.hidePoweredBy({ setTo: 'PHP 4.2.0' }));
+app.use(helmet.xssFilter({setOnOldIE: true}));
+
```
Comment (joordy) [inline server.js:27]: Nice! Good to see that our app has some security 💪

File: view/filter.ejs [hunk 3]
```diff
@@ -55,6 +55,21 @@
...
+                            <%}%>>#fantasyMovies</option>
+                            <option value="thrillerMovies" <%if(movie === "thrillerMovies"){%>
+                                <%="selected"%>
+                            <%}%>>#thrillerMovies</option>
+                            <option value="disneyMovies" <%if(movie === "disneyMovies"){%>
+                                <%="selected"%>
+                            <%}%>>#disneyMovies</option>
+                            <option value="westernMovies" <%if(movie === "westernMovies"){%>
+                                <%="selected"%>
+                            <%}%>>#westernMovies</option>
```
Comment (joordy) [inline view/filter.ejs:72]: Oh god.. I've got stress with choosing my fave one

Comment (veerleprins) [reply view/filter.ejs:72]: 😆

File: routes/profile.js (hunk 1 above)
Comment (veerleprins) [reply routes/profile.js:38]: If I declare it as a variable yes, but not under the variable req.session.user, unfortunately... So something goes wrong somewhere with the req.session

**Pull request merged**